src/components/Header/index.less

```diff
@@ -20,7 +20,7 @@
       justify-content: flex-start;
 
       .logo {
-        height: 100%;
+        height: 80%;
```

Comment:
There is a padding set already. Is this not working as expected?

Reply:
No, it is not: The padding is only applied left and right when increased

Reply:
tried with the margin, same problem. I can however adjust the padding on the parent div instead. Would that be a more suitable solution?